rpm/podman.spec

```diff
@@ -300,6 +300,7 @@ cp -pav test/system %{buildroot}/%{_datadir}/%{name}/test/
 %files docker
 %{_bindir}/docker
 %{_mandir}/man1/docker*.1*
+%{_sysconfdir}/profile.d/%{name}-docker.*
```

Comment:
Shouldn't this be under `%files remote`?

Reply:
No these are only hipped if the user installs podman-docker package.